src/controllers/users.controller.js

```diff
@@ -0,0 +1,81 @@
+const { usersService } = require('./../services');
+
+const getAll = (req, res) => {
+  const users = usersService.getAll();
+
+  res.json(users);
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+
+  const user = usersService.getById(+id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.json(user);
+};
+
+const create = (req, res) => {
+  const { name } = req.body;
```

Comment:
Consider adding validation for the `name` field to ensure it meets any required criteria (e.g., minimum length, allowed characters) before proceeding with user creation.